dashboard/app/controllers/registrations_controller.rb

```diff
@@ -165,6 +228,7 @@ def update_params(params)
       :full_address,
       :terms_of_service_version,
       :provider,
+      :email_opt_in,
```

Comment:
What is this parameter?  Is it related to the commented-out field in https://github.com/code-dot-org/code-dot-org/pull/22471/files#diff-dc1bee46acd2e0072eed26340a1a5845R160 ?

Reply:
:man_facepalming: Yep! I need to remove this.  Nice catch!